server.py

```diff
@@ -35,6 +35,13 @@ def template(tpl, status=200, **vars):
     return 200, headers, src.encode('utf-8')
 
 
+def parse_post_body(env):
```

Comment:
Nice. This makes me wonder if we can normalize the return value a bit more, so that the users don't have to do as much song-and-dance to get their value out. E.g., decode keys to unicode strings, take last value of list by default, and decode the resulting value as well.